docs/src/main/sphinx/connector/kafka.rst

```diff
@@ -25,11 +25,16 @@ needs.
 
 See the :doc:`kafka-tutorial`.
 
-.. note::
+Requirements
+------------
+
+Requirements for using the connector in a catalog to connect to a Kafka
```

Comment:
GDDSD suggests the use of [active voice](https://developers.google.com/style/voice?hl=en):

To connect to a Kafka data source, the following requirements must be met:

Reply:
I like the sentence from @m57lyra  but if we do that we should do it consistently in all connectors. The current one is the established standard for now.